bp_common/test/Makefile

```diff
@@ -84,7 +85,7 @@ opensbi_elf:
 
 bbl_elf:
 	cd $(TEST_DIR)/src/riscv-pk/ && mkdir -p build && cd build \
-	&& ../configure --host=riscv64-unknown-elf --with-arch=rv64ia --with-abi=lp64 --includedir=$(TEST_DIR)/src/riscv-pk/machine/
+	&& ../configure --host=riscv64-unknown-elf --with-arch=rv64ima --with-abi=lp64 --includedir=$(TEST_DIR)/src/riscv-pk/machine/
```

Comment:
Can we just remove the bbl target from this Makefile?
We generate it from the other repo.

Reply:
yep, also will remove opensbi which will be similar